Glossary additions (#286)
BitcoinDesign/Guide: guide/glossary.md

- Bitcoin wallet account
- Service account

### Bitcoin / bitcoin

Bitcoin with a capital *B* is typically associated with Bitcoin the protocol and payment network. It is also often used to refer to as the ecosystem as a whole when writing about it in general terms. Bitcoin with a lowercase “b” written as “bitcoin” is usually associated specifically with bitcoin as the currency.

### Bitcoin Client

Software that runs and/or connects to the Bitcoin network.

### Bitcoin Core

> An open source Bitcoin project that maintains and releases the primary Bitcoin reference implementation the [Bitcoin Core client](bitcoin-core-client).

Reviewer: ```suggestion
An open-source project that maintains and releases the most popular [Bitcoin client](bitcoin-core-client) of the same name.
```

Author: Yep i like it :)

- [Bitcoin Core website](https://bitcoincore.org/)
- [Bitcoin wiki entry](https://en.bitcoin.it/wiki/Bitcoin_Core)

### Bitcoin Core client

> An open source Bitcoin client that is considered the primary reference implementation for Bitcoin. It is the continuation of Satoshi Nakamoto's original bitcoin client released on 9th January 2009. 

Reviewer: ```suggestion
The software which is considered both the reference implementation, as well as the specticiation for the Bitcoin protocol. It is the continuation of Satoshi Nakamoto's original bitcoin client released on 9th January 2009. 
```
